src/applications/letters/components/NoAddressBanner.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
I saw that we have a handful of similar small little jsx chunks in `utils/helpers.jsx` in letters. Could put that there if it makes more sense. I'm generally in favor of a small single use file rather than tacking onto a larger one, but happy to update either way